code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -204,4 +204,163 @@ datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
 	result = "facid"
 	required_reagents = list("sacid" = 1, "fluorine" = 1, "hydrogen" = 1, "potassium" = 1)
 	result_amount = 4
-	required_temp = 380
\ No newline at end of file
+	required_temp = 380
+
+datum/reagent/initropidril
+	name = "Initropidril"
+	id = "initropidril"
+	description = "Major stamina regeneration buff. 33% chance to hit with 5-25 TOX. 5-10% chances to stun and cause suffocation or immediate heart failure."
```

Comment:
I realise I'm saying this a LOT, but it's kinda annoying. I think most of these are just lifted from the .co wiki anyway.